include/aikido/planner/vectorfield/VectorFieldPlanner.hpp

```diff
@@ -83,11 +83,14 @@ aikido::trajectory::UniqueInterpolatedPtr planToEndEffectorOffset(
 /// \param[in] bn Body node of the end-effector.
 /// \param[in] constraint Trajectory-wide constraint that must be satisfied.
 /// \param[in] goalPose Desired end-effector pose.
-/// \param[in] poseErrorTolerance How a planned trajectory is allowed to
+/// \param[in] goalTolerance Distance from goal to stop planning.
+/// \param[in] conversionRatioInGeodesicDistance Conversion ratio from radius to
```

Comment:
It's not computing the goalTolerance. It is computing the geodesic distance from the goal and comparing with goalTolerance.

Reply:
Sure, that was my typo, my core point was that the conversion ratio itself is not being compared to the goalTolerance, it is being used in a computation whose result is compared to goalTolerance. I found this comment confusing, and perhaps something like "Conversion ratio from radius to meter in computing geodesic distance from goal, to be compared with goalTolerance" would be clearer?

(Anyway, this is small enough that I'm fine either way, since looking in the CPP at its usage will quickly clarify it.)

Reply:
Another (perhaps better) way to clarify it would be to change the "goalTolerance" comment to be "Geodesic distance from goal to stop planning" and then the comment on "conversionRatioInGeodesicDistance" could be "Conversion ratio from radius to meter in computing geodesic distance," and it will be implied that it is used in comparison with goal tolerance since you mentioned "geodesic distance" in the "goalTolerance" comment